### DISCUSSION TITLE
add external script source into a blog

### DISCUSSION THREAD

**Discussion by jfvdvelden** [gethinode/hinode#605] [Help]
Hi 2 all,

Second problem:

Formaly i use a simple link to show an external widget inside my blog:

<script src="https://link to source" charset="utf-8"></script>

but now it's not showed any more.

What do i do wrong?

**Comment by markdumay** [comment]
It's the same issue as with the `iframe`, see [this answer](https://github.com/gethinode/hinode/discussions/604#discussioncomment-7473876) for more details.

To help you with debugging - many browsers give you hints when refusing to load stylesheets, scripts, and other files. For example, in Google Chrome you can right-click on a page and select `Inspect` to see any issues.
